apps/ui/src/composables/useActions.ts

```diff
@@ -13,8 +13,12 @@ import type {
   VoteType
 } from '@/types';
 import type { Connector, StrategyConfig } from '@/networks/types';
+import { starknetNetworks } from '@snapshot-labs/sx';
 
 const offchainNetworkId = offchainNetworks.filter(network => enabledNetworks.includes(network))[0];
+const starknetNetworkId = (Object.keys(starknetNetworks) as NetworkID[])
+  .filter(network => enabledNetworks.includes(network))
+  .pop() as NetworkID;
```

Comment:
Why do we `pop` there, but above we just do index access for `[0]`? A bit inconsistent.

Reply:
I am poping to use testnet first ><